draco/analysis/delay.py

```diff
@@ -1182,6 +1266,74 @@ def _draw_ps_sample(d):
     return spec
 
 
+def wiener_filter(
```

Comment:
```suggestion
def delay_spectrum_wiener_filter(
```
For clarity about what the function does and consistency with `delay_spectrum_gibbs`

Reply:
Done. I also renamed `delay_spectrum_gibbs` -> `delay_power_spectrum_gibbs` to distinguish between delay spectra and power spectra, and aliased the former to the latter for backwards-compatibility